### ISSUE TITLE
Radcliffe 2: Subnav items font color contrast and active menu item color contrast

### ISSUE THREAD

**Issue by dhanson-wp** [Automattic/themes#8596]
The font color for Radcliffe 2 theme's Modern Bauhaus style pack is not accessible and is showing inconsistent display across browsers.

When you hover over subnav menu items, the font color changes to `#fff` on a `#fff` background and is not visible to the user. This is seen in Chrome, Arc, and Firefox browsers when logged in and on Safari not logged in.

![Image](https://github.com/user-attachments/assets/8326c382-4c6b-4bd1-9943-e9277e61590b)

In the Customizer, you can see the font's hover color `#939393` as expected.

![Image](https://github.com/user-attachments/assets/b587dfbc-1861-47be-a1fb-f09fd4cb8c3b)

Subnav menu items that are visible are a gray color value `#939393` on `#fff` background on hover. Active menu items have the same color combination.

The behavior is inconsistent, and the root cause is unclear. 

### Recommendations

Change the hover color and active menu item color from `#939393` to something more readable on the `#fff` background to significantly improve the readability.

Fix the inconsistent hover style behavior for logged in and logged out users.
